### ISSUE TITLE
PyPi library

### ISSUE THREAD

**Issue by dpoursanidis** [BrandonSmithJ/MDN#2]
Are plans to add the MDN package in the PyPi libary for easier use?

**Comment by BrandonSmithJ** [comment]
Yes, thanks for the suggestion. We have a paper being prepared right now, which should be submitted in a couple weeks. At that point I'll be releasing a new version and can see about getting it into PyPi.

**Comment by dpoursanidis** [comment]
Great! Let me know when it will be in PyPi.
Good luck also for the new paper :)

**Comment by geo-dingky** [comment]
Hello,
How can I use my own data to train the model

**Comment by BrandonSmithJ** [comment]
@geo-dingky See my reply to your prior issue: https://github.com/BrandonSmithJ/MDN/issues/6#issuecomment-846194384

Please reply on that issue if you have any questions about it.

**Comment by geo-dingky** [comment]
Sincerely thank you for your help，

I try your suggestion, unfortunately I get the following error message：
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-4ad565c9c3e4> in <module>
----> 1 from MDN.utils import get_args
      2 from MDN.product_estimation import get_estimates
      3 import numpy as np
      4 

ImportError: cannot import name 'get_args' from 'MDN.utils' (C:\Users\DELL\002file\MDN\utils.py)

I didn't find  'get_args'  in the utils.py

**Comment by BrandonSmithJ** [comment]
@geo-dingky get_args is actually in parameters, and so it's "from MDN.parameters import get_args", sorry for the confusion. Please reply to the other issue [#6](https://github.com/BrandonSmithJ/MDN/issues/6#issuecomment-846194384) so that I get an email notification, rather than editing prior comments.
